src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/ThemeMode.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.ComponentModel;
+
+using System.ComponentModel.Design.Serialization;
+using System.Diagnostics;
+using System.Globalization;
+using System.Reflection;
+using System.Windows;
+using System.Windows.Markup;
+using System.Security;
+using MS.Internal;
+using MS.Utility;
+using System.Diagnostics.CodeAnalysis;
+
+namespace System.Windows
+{
+    [Experimental("WPF0001")]
+    public readonly struct ThemeMode : IEquatable<ThemeMode>
+    {
+        public static ThemeMode None => new ThemeMode();
+        public static ThemeMode Light => new ThemeMode("Light");
+        public static ThemeMode Dark => new ThemeMode("Dark");
+        public static ThemeMode System => new ThemeMode("System");
+
+
+        public string Value => _value ?? "None";
+
+        public ThemeMode(string value) => _value = value;
```

Comment:
Sorry for the late review but should the constructor really be public ? Do we expect developers to be able to declare there own `ThemeMode` ?

The code also seems to restrict the supported ThemeMode here: https://github.com/dipeshmsft/wpf/blob/998c4a452f2882356e690ab6c7669194cdaa89ec/src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/ThemeManager.cs#L199

Reply:
Doesn't mean the type cannot be used anywhere else...

Think of it as a string with autocomplete support. It's experimental anyway and I am hoping it will go away in the future.

Reply:
@ThomasGoulet73, although your point is right here, and doing this will reduce the usage of this API by developers, but I will let it be for now as this is already marked experimental.